quill-sql/src/main/scala/io/getquill/source/sql/idiom/SqlIdiom.scala

```diff
@@ -162,8 +162,10 @@ trait SqlIdiom {
   implicit def orderByCriteriaShow(implicit strategy: NamingStrategy): Show[OrderByCriteria] = new Show[OrderByCriteria] {
     def show(criteria: OrderByCriteria) =
       criteria match {
-        case OrderByCriteria(prop, true)  => s"${prop.show} DESC"
-        case OrderByCriteria(prop, false) => prop.show
+        case OrderByCriteria(prop, AscNullsFirst)  => s"${prop.show} ASC NULLS FIRST"
```

Comment:
I'm unsure what's better here. I like having the same behavior regardless of the database used, but also I feel that it should be possible to obtain a query that doesn't contain the NULLS FIRST/NULLS LAST included.

Currently, ```Ord.asc``` and ```Ord.ascNullsFirst``` generate the same query, both generate ```ASC NULLS FIRST```. What do you think about having ```Ord.asc``` not generate the NULLS FIRST part (essentially having a nulls default behavior), but keep ```Ord.ascNullsFirst`` as default if no ordering is specified?